kANe/v1p2.md

```diff
@@ -20354,6 +20478,8 @@ ___Paroda O. I. No. 8354. त्तिक rites of वर्णs and आश्
 
 विश्वम्भर. 
 
+## Appendix B
```

Comment:
What should be the title here 'Appendix B' or 'List of Authors on Dharmaśāstra'?